site/styles/main.scss

```diff
@@ -52,7 +52,10 @@ html {
   overflow-y: scroll;
 }
 body {
-  font-family: "Rawline", $font-family-base;
+	font-family: "Rawline", $font-family-base;
+	min-height: 100vh;
```

Comment:
[Apparently there is an IE11 `min-height` bug](https://github.com/philipwalton/solved-by-flexbox/blob/master/assets/css/components/site.css#L2), so [potentially should just be `height`](https://github.com/philipwalton/solved-by-flexbox/blob/master/assets/css/components/site.css#L12) - has this been tested there?

Reply:
Yeah. The `min-height:` does not work on IE11. By using `height: 100%`, you also need to add `height: 100%` to the html element. This is because the body relies on a parent for the height. In addition, replacing `min-height` with `height` also means that the body doesn't extend down the page anymore.

![image](https://user-images.githubusercontent.com/41402426/100794010-d9891f80-3481-11eb-8ad8-3745e3e52230.png)

Is this important to consider? or is it a non-issue?

Reply:
I guess the important factor is, if `min-height` doesn't work on IE11, does that mean the sticky footer doesn't work on IE11?

Reply:
The sticky footer does not work in IE11 with `min-height` but it does work with `height`. I am going to use the latter method.